.github/workflows/style.yaml

```diff
@@ -0,0 +1,53 @@
+name: Style
+on:
+  pull_request:
+    types:
+    - opened
+    - synchronize
+    - labeled
```

Comment:
`reopened` is missing.

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.

From https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request